infra/aws-elb-503-subscription-endpoint/templates/ingress.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   namespace: default
   name: 503-for-pending-subscriptions
   annotations:
-    alb.ingress.kubernetes.io/certificate-arn: >-
-      arn:aws:acm:eu-west-1:242905224710:certificate/de2efba9-8964-49d2-8525-242e7e43ca4c
+    alb.ingress.kubernetes.io/certificate-arn: >- 
+      {{ .Values.myAccount.acmCertificate }}
```

Comment:
I am not really sure whether this change is ok, I copied how this was set in `charts/nodejs/templates/ingress.yaml` but would like to go over it with someone to make sure it is ok

Reply:
yup, this is correct. This should have been fixed before, there was a ticket for it. I am surprised that we skipped it. Thank you for finding it!